### PR TITLE
Self-Help Commands Init Onboarding

### PR DESCRIPTION
**Pull request by BharathaAravind**
---
Init PR to add self-help commands for Azure Help RP.

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by BharathaAravind** [comment]
@yonzhan, @kairu-ms, @necusjz, @jsntcy - The pr is ready for review. I added tests and recording files to the pr for review.

File: src/self-help/azext_self_help/aaz/latest/self_help/diagnostic/_create.py [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "self-help diagnostic create",
+    is_preview=True,
+)
+class Create(AAZCommand):
+    """Creates a diagnostics that will help you troubleshoot an issue with your azure resource.
+    """
```
Comment (kairu-ms) [inline src/self-help/azext_self_help/aaz/latest/self_help/diagnostic/_create.py:20]: Could you add examples in aaz-dev-tools and regenerate the code?

Comment (BharathaAravind) [reply src/self-help/azext_self_help/aaz/latest/self_help/diagnostic/_create.py:20]: I actually added examples for all the commands. I do not see any changes pending in my local.

Comment (BharathaAravind) [reply src/self-help/azext_self_help/aaz/latest/self_help/diagnostic/_create.py:20]: The examples seems to be in aaz repo.  
https://github.com/Azure/aaz/pull/176/commits/c7c6c34e81c73b9373908ae43c199959142ca27f

PR for AAZ repo: https://github.com/Azure/aaz/pull/176

**Pull request closed without merging**